lib/seven_segment.ex

```diff
@@ -0,0 +1,164 @@
+defmodule SS2 do
+
+  def digitals(["a", "b", "c", "e", "f", "g"]), do: 0
```

Comment:
```suggestion
  def digitals(~w[a b c e f g]), do: 0  
```
А еще лучше сразу работать с чарлистами, а ввод сразу к ним преобразовать
```elixir
  def digitals('abcefg'), do: 0
```